content/docs/intro/cloud-providers/kubernetes/setup.md

```diff
@@ -12,7 +12,7 @@ aliases: ["/docs/reference/clouds/kubernetes/setup/"]
 [aks-tutorial]: {{< relref "/docs/tutorials/kubernetes/aks" >}}
 [Heptio AWS quickstart]: https://aws.amazon.com/quickstart/architecture/heptio-kubernetes/
 [provider-args]: {{< relref "/docs/reference/pkg/kubernetes/provider" >}}
-[provider-kubeconfig]: {{< relref "/docs/reference/pkg/kubernetes/provider#resource-arguments" >}}
+[provider-kubeconfig]: {{< relref "/docs/reference/pkg/kubernetes/provider#inputs" >}}
 [kubeconfig]: https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/
 [install]: {{< relref "/docs/get-started/install" >}}
 [nodejs]: https://nodejs.org/en/
```

Comment:
line 14 and 15 are linking to essentially the same place, should they be combined?